src/connector/src/sink/mysql.rs

```diff
@@ -125,8 +149,57 @@ impl TryFrom<Datum> for MySQLValue {
     }
 }
 
+#[allow(clippy::upper_case_acronyms)]
+#[derive(strum_macros::Display)]
```

Comment:
what does `strum_macros::Display` do

Reply:
proc_macro Display converts enum variants to strings, i.e. `MySQLDataType::BOOL` will be printed as `"BOOL"`. In this case I used it for the schema in the create table statement.